skrifa/src/outline/glyf/hint/graphics_state/mod.rs

```diff
@@ -0,0 +1,211 @@
+#[derive(Copy, Clone, Debug)]
+pub struct RetainedGraphicsState {
+    /// Controls whether the sign of control value table entries will be
+    /// changed to match the sign of the actual distance measurement with
+    /// which it is compared.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#auto%20flip>
+    pub auto_flip: bool,
+    /// Limits the regularizing effects of control value table entries to
+    /// cases where the difference between the table value and the measurement
+    /// taken from the original outline is sufficiently small.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#control_value_cut-in>
+    pub control_value_cutin: i32,
+    /// Establishes the base value used to calculate the range of point sizes
+    /// to which a given DELTAC[] or DELTAP[] instruction will apply.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#delta%20base>
+    pub delta_base: u16,
+    /// Determines the range of movement and smallest magnitude of movement
+    /// (the step) in a DELTAC[] or DELTAP[] instruction.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#delta%20shift>
+    pub delta_shift: u16,
+    /// Makes it possible to turn off instructions under some circumstances.
+    /// When set to TRUE, no instructions will be executed
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#instruct%20control>
+    pub instruct_control: u8,
+    /// Establishes the smallest possible value to which a distance will be
+    /// rounded.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#minimum%20distance>
+    pub min_distance: i32,
+    /// Determines whether the interpreter will activate dropout control for
+    /// the current glyph.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#scan%20control>
+    pub scan_control: bool,
+    /// Type associated with `scan_control`.
+    pub scan_type: i32,
+    /// The distance difference below which the interpreter will replace a
+    /// CVT distance or an actual distance in favor of the single width value.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#single_width_cut_in>
+    pub single_width_cutin: i32,
+    /// The value used in place of the control value table distance or the
+    /// actual distance value when the difference between that distance and
+    /// the single width value is less than the single width cut-in.
+    ///
+    /// See <https://developer.apple.com/fonts/TrueType-Reference-Manual/RM04/Chap4.html#single_width_value>
+    pub single_width: i32,
+}
+
+impl Default for RetainedGraphicsState {
+    fn default() -> Self {
```

Comment:
Is there a reference for where the defaults came from?

Reply:
Added links to graphics state summary table in the spec which lists the default values.